cli/src/main.rs

```diff
@@ -32,17 +32,26 @@ const HELP: &str = "
 ";
 
 fn update(client: &mut user::User, group_id: Option<String>, stdout: &mut StdoutLock) {
-    let messages = client.update(group_id).unwrap();
-    stdout.write_all(b" >>> Updated client :)\n").unwrap();
-    if !messages.is_empty() {
-        stdout.write_all(b"     New messages:\n\n").unwrap();
+    let messages = client.update(group_id);
```

Comment:
This is unrelated to k256 support but added some basic error handling so that the CLI wouldn't panic and crash on update issues.